### PR TITLE
[DGPFET-10] The Software Developer conditionals

### PR DESCRIPTION
**Pull request by Ig-Matrix**

### CODE REVIEW

File: submissions/ighawosaOmoma/Javascript/conditionals.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// })
+for(var i=1; i<positiveDifference; i++){
+    if(i % 3 ===0 && i % 5 ===0){
+        document.write('Software Developer <br>')
+    }
+    else if (i % 3 ===0){
+        document.write('Software <br>')
+    }
+    else if(i % 5 ===0){
+        document.write('Developer <br>')
```
Comment (Mayowa2020) [inline submissions/ighawosaOmoma/Javascript/conditionals.js:13]: This line is not well concatenated. Please , insert a plus sign in between Developer and br just like you did in line 15 and put the quotes appropriately. Do the same for lines 7 & 10.  Run your code and attach the a Screenshot of your output to this PR. Kindly tag me on Slack @Bukky, once you are done.

**Review by Mayowa2020 — CHANGES_REQUESTED** [review]
Well done @Ig-Matrix ! KIndly effect the requested changes and revert.

**Comment by Ig-Matrix** [comment]
Screenshot (135)
Screenshot (136)

**Review by Mayowa2020 — APPROVED** [review]
Well done @Ig-Matrix ! Approved...

**Pull request merged**